content/en/observability_pipelines/setup/datadog_with_archiving.md

```diff
@@ -0,0 +1,529 @@
+
+{{% /tab %}}
+{{% tab "Terraform (AWS)" %}}
+
+{{% op-datadog-archives-s3-setup %}}
+
+3. Attach the policy to the IAM Instance Profile that is created with Terraform, which you can find under the `iam-role-name` output.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Connect the S3 bucket to Datadog Log Archives
+
+You need to connect the S3 bucket you created earlier to Datadog Log Archives so that you can rehydrate the archives later on.
+
+1. Navigate to Datadog [Log Forwarding][5].
+1. Click **+ New Archive**.
+1. Enter a descriptive archive name.
+1. Add a query that filters out all logs going through log pipelines so that those logs do not go into this archive. For example, add the query `observability_pipelines_read_only_archive`, assuming that no logs going through the pipeline have that tag added.
+1. Select **AWS S3**.
+1. Select the AWS Account that your bucket is in.
+1. Enter the name of the S3 bucket.
+1. Optionally, enter a path.
+1. Check the confirmation statement.
+1. Optionally, add tags and define the maximum scan size for rehydration. See [Advanced settings][6] for more information.
+1. Click **Save**.
+
+See the [Log Archives documentation][7] for additional information.
+
+## Install the Observability Pipelines Worker
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+
+The Observability Pipelines Worker Docker image is published to Docker Hub [here][1].
+
+1. Download the [sample pipeline configuration file][2].
+
+2. Run the following command to start the Observability Pipelines Worker with Docker:
+
+    ```shell
+    docker run -i -e DD_API_KEY=<API_KEY> \
+      -e DD_OP_PIPELINE_ID=<PIPELINE_ID> \
+      -e DD_SITE=<SITE> \
+      -e AWS_ACCESS_KEY_ID=<AWS_ACCESS_KEY_ID> \
+      -e AWS_SECRET_ACCESS_KEY=<AWS_SECRET_ACCESS_KEY> \
+      -p 8282:8282 \
+      -v ./pipeline.yaml:/etc/observability-pipelines-worker/pipeline.yaml:ro \
+      datadog/observability-pipelines-worker run
+    ```
+    
+    Replace `<API_KEY>` with your Datadog API key, `<PIPELINES_ID>` with your Observability Pipelines configuration ID, and `<SITE>` with {{< region-param key="dd_site" code="true" >}}. Replace `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` with the AWS credentials you created earlier. `./pipeline.yaml` must be the relative or absolute path to the configuration you downloaded in Step 1.
+  
+[1]: https://hub.docker.com/r/datadog/observability-pipelines-worker
+[2]: /resources/yaml/observability_pipelines/archives/pipeline.yaml
+{{% /tab %}}
+{{% tab "AWS EKS" %}}
+1. Download the [Helm chart][1] for AWS EKS.
+
+2. In the Helm chart, replace the `datadog.apiKey` and `datadog.pipelineId` values to match your pipeline and use {{< region-param key="dd_site" code="true" >}} for the `site` value. Replace `${DD_ARCHIVES_SERVICE_ACCOUNT}` in the Helm config with the service account you created earlier. Then, install it in your cluster with the following commands:
```

Comment:
Could you add the following sentences in [this screenshot](https://a.cl.ly/KouN5YB0)? These values also need to be replaced in the helm chart

Also, let's turn the last sentence that starts with `then` into step 3

Reply:
Also linking you the [Figma design](https://www.figma.com/file/G6kPDXDo6aaBQvjaDpKpXG/Use-Case-Informed-Onboarding?type=design&node-id=1597-39264&mode=design&t=Jp6Mi0KFxHRBLhtl-0) with the copies